package/boot/uboot-layerscape/files/fsl_ls1046a-frwy-uEnv.txt

```diff
@@ -0,0 +1,8 @@
+fdtaddr=0x8f000000
+loadaddr=0x81000000
+bootm_size=0x10000000
+hwconfig=fsl_ddr:bank_intlv=auto
+qspi_boot=sf probe 0:0;sf read $fdtaddr f00000 100000;sf read $loadaddr 1000000 1000000;bootm $loadaddr - $fdtaddr
+bootargs=root=/dev/mtdblock9 rootfstype=squashfs,jffs2 noinitrd earlycon=uart8250,mmio,0x21c0500 console=ttyS0,115200 mtdparts=1550000.spi:1m(bl2),4m(fip),1m(u-boot-env),3m(reserved-1),256k(fman),5888k(reserved-2),1m(dtb),16m(kernel),32m(rootfs),48m@0x1000000(firmware)
```

Comment:
Shouldn't that be 49m@0xf00000 due to dtb inclusion in image?
@yangbolu1991 @CHKDSK88 
Please just answer here and do not update/repush, as I've already picked stuff.

Reply:
It should be `49m@0xf00000`. Now sysupgrade for this devices are broken.